src/server.rs

```diff
@@ -3627,4 +3798,89 @@ csv.from(file: "my.csv")
             panic!("Result was not a token result");
         }
     }
+
+    #[test]
+    async fn test_code_action_import_insertion() {
```

Comment:
Per my comment above about the `package` clause can you add a test case where the original source code contains a `package` clause?

Reply:
Test added.